pkg/apply/apply_test.go

```diff
@@ -51,9 +50,32 @@ func TestApplyBasicUpdates(t *testing.T) {
 	assert.Equal(t, int64(2000000), applier.throttleBytes)
 
 	defer applier.adminClient.Close()
-	err := applier.Apply(ctx)
+	changes, err := applier.Apply(ctx)
 	require.NoError(t, err)
 
+	// test ChangesTracker has expected shape
+	expectedChanges := &ChangesTracker{
```

Comment:
You are asserting the result only in the case where a new topic is created. Please add an assertion when the topic is updated.